js/start.md

```diff
@@ -501,6 +501,7 @@ export default App;
 Next, update `src/App.js` to list all the items in the database by importing `listTodos` and then using [Hooks](https://reactjs.org/docs/hooks-intro.html) to update the page when a query runs on app start by adding initial state and a `reducer` function as well as modifying your `App` function:
 
 ```javascript
+import React, { useEffect, useReducer } from 'react' // using hooks
```

Comment:
```suggestion
import { useEffect, useReducer } from 'react' // using hooks
```

I know it could be on one line but if you are adding code on your app, specially for people that is starting on JS it will be simpler to just add this after `// other imports`

Reply:
I agree! Updated based on your comment.